test/functional/test_host_io.py

```diff
@@ -111,7 +111,7 @@ def prepare_ssh(self):
 
     def setUp(cls):
         DOWNLOAD_URL = 'https://github.com/InfraSIM/test/raw/master/image/kcs.img'
-        MD5_KCS_IMG = 'cfdf7d855d2f69c67c6e16cc9b53f0da'
+        MD5_KCS_IMG = '986e5e63e8231a307babfbe9c81ca210'
```

Comment:
@JuneZhou19 it would be necessary to update the image to above `DOWNLOAD_URL` before this PR